README.md

```diff
@@ -238,6 +239,31 @@ var errorCallback = function () {
 company.addRecord('sales', sale, successCallback, errorCallback);
 ```
 
+### Treasure#fetchGlobalID(success, failure, forceFetch)
```

Comment:
I think it's more common to do params first and callbacks last. This ordering seems weird to me.